src/test/java/BaseTest.java

```diff
@@ -9,18 +9,38 @@
 import org.testng.annotations.BeforeSuite;
 import org.testng.annotations.Parameters;
 
-import java.time.Duration;
-
-
-public class BaseTest {
-    static WebDriver driver;
-    public String url = null;
-
-
     @BeforeSuite
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/217740879-66080110-6b5b-452f-86d1-ec90d19a7d62.png)